networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -100,10 +95,6 @@ impl SuzukaPartialNode<Executor> {
 		let commitment_events =
 			if config.mcr.should_settle() { Some(commitment_events) } else { None };
 
-		debug!("Creating the movement rest service");
-		let movement_rest =
-			MovementRest::try_from_env().context("Failed to create MovementRest")?;
-
```

Comment:
I think this should be reverted, as explained in other comments.

Reply:
It's not used anymore. The Rest service isn't created using this struct, so I remove it.

Reply:
I mean, it should be instantiated and used as it previously was.

Reply:
I think it's not used and as we remove the finality entry point, we can remove everything. The services are started with opt and fin view. This rest service bind port is hard coded, and I see nowhere where it's used. So I think it has been added but never used.

Reply:
I removed the MovementRest crate.